brainpy/_src/math/compat_pytorch.py

```diff
@@ -86,3 +86,145 @@ def unsqueeze(input: Union[jax.Array, Array], dim: int) -> Array:
   """
     input = _as_jax_array_(input)
     return Array(jnp.expand_dims(input, dim))
+
+
+# Math operations
+def abs(input: Union[jax.Array, Array],
+        *, out: Optional[Union[Array, jax.Array, np.ndarray]] = None) -> Optional[Array]:
+  input = _as_jax_array_(input)
+  r = jnp.abs(input)
+  if out is None:
+    return _return(r)
+  else:
+    _check_out(out)
+    out.value = r
+
+absolute = abs
+
+def acos(input: Union[jax.Array, Array],
+         *, out: Optional[Union[Array,jax.Array, np.ndarray]] = None) -> Optional[Array]:
+  input = _as_jax_array_(input)
+  r = jnp.arccos(input)
+  if out is None:
+    return _return(r)
+  else:
+    _check_out(out)
+    out.value = r
+
+arccos = acos
+
+def acosh(input: Union[jax.Array, Array],
+          *, out: Optional[Union[Array,jax.Array, np.ndarray]] = None) -> Optional[Array]:
+  input = _as_jax_array_(input)
+  r = jnp.arccosh(input)
+  if out is None:
+    return _return(r)
+  else:
+    _check_out(out)
+    out.value = r
+
+arccosh = acosh
+
+def add(input: Union[jax.Array, Array, jnp.number],
+        other: Union[jax.Array, Array, jnp.number],
+        *, alpha: Optional[jnp.number] = 1,
```

Comment:
where is `alpha` used?

Reply:
In Pytorch, the `add` func is not only "add". It also means "Adds other, scaled by alpha, to input.", and `alpha` is used in line 134.